src/Sulu/Bundle/AdminBundle/Resources/js/containers/FieldBlocks/tests/FieldBlocks.test.js

```diff
@@ -898,6 +901,130 @@ test('Should call onFinish when the order of the blocks has changed', () => {
+                        },
+                    },
+                },
+                position_left: {
+                    label: 'Position Left',
+                    type: 'single_select',
+                    options: {
+                        default_value: {
+                            name: 'default_value',
+                            type: 'string',
+                            value: 'left',
+                        },
+                        values: {
+                            name: 'values',
+                            type: 'collection',
+                            value: [
+                                {
+                                    name: 'left',
+                                    title: 'Left',
+                                },
+                                {
+                                    name: 'center',
+                                    title: 'Center',
+                                },
+                                {
+                                    name: 'right',
+                                    title: 'Right',
+                                },
+                            ],
+                        },
+                    },
+                },
+                position_right: {
+                    label: 'Position Right',
+                    type: 'single_select',
+                    options: {
+                        default_value: {
+                            name: 'default_value',
+                            type: 'string',
+                            value: 'right',
+                        },
+                        values: {
+                            name: 'values',
+                            type: 'collection',
+                            value: [
+                                {
+                                    name: 'left',
+                                    title: 'Left',
+                                },
+                                {
+                                    name: 'center',
+                                    title: 'Center',
+                                },
+                                {
+                                    name: 'right',
+                                    title: 'Right',
+                                },
+                            ],
+                        },
+                    },
+                },
+            },
+        },
+    };
+
+    formInspector.getSchemaEntryByPath.mockReturnValue({types});
+
+    fieldRegistry.get.mockReturnValue(SingleSelect);
+
+    const changeSpy = jest.fn();
+
+    const fieldBlocks = mount(
+        <FieldBlocks
+            {...fieldTypeDefaultProps}
+            defaultType="default"
+            formInspector={formInspector}
+            minOccurs={1}
+            onChange={changeSpy}
+            types={types}
+            value={observable([{type: 'default'}])}
+        />
+    );
+
+    fieldBlocks.find('Block').at(0).simulate('click');
```

Comment:
Out of curiosity: Is that click really necessary? Shouldn't the call below happen anyway? Because if it doesn't, there might be some edge case, which might cause some weird behavior.

Reply:
No the call does not happen. The blocks are collapsed and the rendering is only called after they are expanded.

Reply:
Ok, when a new block is added, it is already opened anyway... So that should not be that bad. I guess the only thing that might not work as expected is if a new field is added to a block type, which is already used somewhere. The value would probably not be updated, until the block is expanded in the UI. But I think that this is an edge case, which does not hurt.